Docs/build.ps1

```diff
@@ -47,7 +47,7 @@ try {
 Set-Location $BaseDir
 dotnet tool run docfx docfx.json @args
 
-if (-not (Test-Path "$BaseDir/_site/api/Libplanet.html")) {
+if (-not (Test-Path "$BaseDir/_site/api/index.html")) {
```

Comment:
Since the `Libplanet` namespace was gone, it should update `docs/api/index.md` like

```
<meta http-equiv="refresh" content="0; url=Libplanet.Common.html">
Move to [Libplanet.Common](Libplanet.Common.yml)&hellip;
```

And also `docs/toc.yml`

```#- name: Articles
#  href: articles/
- name: Overview
  href: articles/overview.md
- name: Design
  href: articles/design.md
- name: API Reference
  href: api/
  homepage: api/Libplanet.Common.yml
- name: Analyzer
  href: ../Libplanet.Analyzers/
  homepage: ../Libplanet.Analyzers/README.md
- name: CLI Tools
  href: ../Libplanet.Tools/README.md
- name: Explorer
  href: ../Libplanet.Explorer/
  homepage: ../Libplanet.Explorer/README.md
- name: Changelog
  href: ../CHANGES.md
- name: Contribute
  href: ../CONTRIBUTING.md
```

Reply:
Can you make an issue about this?

Reply:
I opened a new issue and I'll mark this conversation as resolved.